Scripts/models/logit.py

```diff
@@ -62,19 +62,19 @@ def _calc_dest_util(self, mode, impedance):
     def _calc_sec_dest_util(self, mode, impedance, orig, dest):
         b = self.dest_choice_param[mode]
         utility = numpy.zeros_like(next(iter(impedance.values())), self.dtype)
-        utility += self._add_sec_zone_util(utility, b["attraction"], orig, dest)
-        utility += self._add_impedance(utility, impedance, b["impedance"])
+        self._add_sec_zone_util(utility, b["attraction"], orig, dest)
+        self._add_impedance(utility, impedance, b["impedance"])
         dest_exps = numpy.exp(utility)
         size = numpy.zeros_like(utility)
         size += self._add_sec_zone_util(size, b["size"])
```

Comment:
This is still done in Johanna's style though?

Reply:
Checked this one from commits and that's correct. I already merged this pull request, so it seems that I need to do another pull request to fix this.